### PR TITLE
DEX-466 ensure individual default name on creation

### PR DESCRIPTION
**Pull request by colinwkingen**
Small PR against EDM work. Creates test to make sure a default name is set on individual creation even if one isn't specified.

### CODE REVIEW

File: tests/modules/individuals/resources/test_individual_names.py [hunk 1]
```diff
@@ -5,11 +5,11 @@
 from tests.modules.individuals.resources import utils as individual_utils
 from tests.modules.sightings.resources import utils as sighting_utils
 
+from app.modules.encounters.models import Encounter
```
Comment (hwindsor) [inline tests/modules/individuals/resources/test_individual_names.py:8]: I think that having models included at the top level in test files has caused problems in the past with circular inheritance. 
I don't have any evidence to back this up but I have a vague memory that it's caused us problems in the past so I avoid it now.

File: tests/modules/individuals/resources/test_individual_names.py [hunk 2]
```diff
@@ -117,3 +121,63 @@ def test_get_set_individual_names(db, flask_app_client, researcher_1):
         )
         sighting.delete_cascade()
         enc.delete_cascade()
+
+
+def test_ensure_default_name_on_individual_creation(db, flask_app_client, researcher_1):
```
Comment (hwindsor) [inline tests/modules/individuals/resources/test_individual_names.py:126]: I'm now trying to use the request.addfinalizer construct to make tests more readable for the cleanup.

**Review by hwindsor — APPROVED** [review]
LGTM Minor comments on the test addition

File: tests/modules/individuals/resources/test_individual_names.py (hunk 1 above)
Comment (naknomum) [reply tests/modules/individuals/resources/test_individual_names.py:8]: interesting!  noted.   <fear>

**Review by naknomum — APPROVED** [review]
looks good

File: tests/modules/individuals/resources/test_individual_names.py (hunk 1 above)
Comment (colinwkingen) [reply tests/modules/individuals/resources/test_individual_names.py:8]: Yes, this was the case for a while but Karen did some work to fix the behavior.

File: tests/modules/individuals/resources/test_individual_names.py (hunk 2 above)
Comment (colinwkingen) [reply tests/modules/individuals/resources/test_individual_names.py:126]: Yes, I forgot about that. Will try to use in the future.

**Pull request merged**